src/main/webapp/scripts/comments.js

```diff
@@ -0,0 +1,25 @@
+export { makeApiCall, buildRequestBody };
```

Comment:
Nit: I believe exports are usually found in the bottom of the file.

Reply:
Import at the beginning and export at the end I see I see. Done